lib/Controller/ChatController.php

```diff
@@ -551,9 +551,11 @@ public function deleteMessage(int $messageId): DataResponse {
 		}
 
 		$attendee = $this->participant->getAttendee();
-		if (!$this->participant->hasModeratorPermissions(false)
-			&& ($message->getActorType() !== $attendee->getActorType()
-				|| $message->getActorId() !== $attendee->getActorId())) {
+		$isOwnMessage = $message->getActorType() === $attendee->getActorType()
+			|| $message->getActorId() === $attendee->getActorId();
```

Comment:
shouldn't this be `&&` ?

Reply:
yeah the integration test also failed, it works now :D